src/common/modals/Settings/components/MyAccount_preferences.js

```diff
@@ -137,7 +142,7 @@ export default function MyAccountPreferences() {
   const currentAccount = useSelector(_getCurrentAccount);
   const releaseC = useSelector(state => state.settings.releaseChannel);
 
-  const [releaseChannel, setReleaseChannel] = useState(0);
+  const [releaseChannel, setReleaseChannel] = useState(releaseC);
```

Comment:
It doesn't make sense to have a local state when you already have this data from redux